src/devices/realsense2/realsense2Driver.cpp

```diff
@@ -339,6 +339,16 @@ static size_t bytesPerPixel(const rs2_format format)
     return bytes_per_pixel;
 }
 
+static YarpDistortion rsDistToYarpDist(const rs2_distortion dist)
+{
+    switch (dist) {
+    case RS2_DISTORTION_BROWN_CONRADY:
+        return YarpDistortion::YARP_PLUM_BOB;
+    default:
+        return YarpDistortion::YARP_DISTORTION_NONE;
```

Comment:
I don't think that the `defualt` should be `YarpDistortion::YARP_DISTORTION_NONE`. The `switch` can reach `default` even if the distortion is different than `RS2_DISTORTION_BROWN_CONRADY` and is not `RS2_DISTORTION_NONE`.

`rs2_distortion` can be:
1. `RS2_DISTORTION_NONE`: Rectilinear images. No distortion compensation required.
2. `RS2_DISTORTION_MODIFIED_BROWN_CONRADY`: Equivalent to Brown-Conrady distortion, except that tangential distortion is applied to radially distorted points.
3. `RS2_DISTORTION_INVERSE_BROWN_CONRADY`: Equivalent to Brown-Conrady distortion, except undistorts image instead of distorting it.
4. `RS2_DISTORTION_FTHETA`: F-Theta fish-eye distortion model.
5. `RS2_DISTORTION_BROWN_CONRADY`: Unmodified Brown-Conrady distortion model.
6. `RS2_DISTORTION_COUNT`: Number of enumeration values. Not a valid input: intended to be used in for-loops.

I guess this is something unimplemented, but `YarpDistortion::YARP_DISTORTION_UNKNOWN` would be more correct (unless `YarpDistortion::YARP_DISTORTION_NONE` means it).
Can we do something about it?